cmd/wallet-sdk-gomobile/api/api.go

```diff
@@ -23,22 +23,23 @@ type JSONArray struct {
 
 // KeyHandle represents a public key with associated metadata.
 type KeyHandle struct {
-	PubKey []byte `json:"key,omitempty"` // Raw bytes
-	KeyID  string `json:"keyID,omitempty"`
+	PubKey []byte      `json:"key,omitempty"` // JWK bytes
+	JWK    *JSONWebKey `json:"jwk,omitempty"`
+	KeyID  string      `json:"keyID,omitempty"`
 }
 
 // KeyWriter represents a type that is capable of performing operations related to key creation and storage within
 // an underlying KMS.
 type KeyWriter interface {
 	// Create creates a keyset of the given keyType and then writes it to storage.
-	// The keyID and raw public key bytes of the newly generated keyset are returned via the KeyHandle object.
+	// The keyID and public key JWK bytes of the newly generated keyset are returned via the KeyHandle object.
```

Comment:
Should change `JWK bytes` to just `JWK` since we're no longer returning bytes.

(There are a few other places in this PR where the language should be updated too)